core/lib/generators/solidus/install/install_generator/install_frontend.rb

```diff
@@ -30,7 +30,14 @@ def install_solidus_frontend
           end
         end
 
-        @generator_context.generate("solidus_frontend:install #{@generator_context.options[:auto_accept] ? '--auto-accept' : ''}")
+        # Solidus bolt will be handled in the installer as a payment method.
+        begin
+          skip_solidus_bolt = ENV['SKIP_SOLIDUS_BOLT']
+          ENV['SKIP_SOLIDUS_BOLT'] = 'true'
```

Comment:
If we move it here, I think we should remove the option from solidus_frontend altogether. Otherwise we're dealing with unneeded complexity here. WDYT?